### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rodrigobsm**
This is a Pull Request as part of the Mobile Code Challenge.

### CODE REVIEW

**Comment by arctouch-victorbotamedi** [comment]
There is a lot of mixed spaces and tabs. Can you fix that?

**Comment by rodrigobsm** [comment]
Hello. I'm sorry. I've fixed the code now, as requested. Please feel free to reply on any point. Thank you.

File: MoviesTips/Views/MoviesList.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            // Binds with ModelView class.
+            this.BindingContext = this._viewModel;
+
+            // Show movie info view when list view item is tapped.
+            this.MoviesListView.ItemTapped += async (sender, e) =>
+            {
+                var Result = e.Item as Models.Result;
+
+                // Call API to get this movie information
+                var Movie = await TMDBAPI.API.GetMovieInfoAsync(Result.Id);
```
Comment (arctouch-victorbotamedi) [inline MoviesTips/Views/MoviesList.xaml.cs]: As the API call is made before pushing the screen, there is a little delay when the user presses the list item.

File: MoviesTips/ViewModels/MoviesSearchViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(results)));
+            }
+        }
+
+        public MoviesSearchViewModel()
+        {
+            // Bind the results to ListView.
+            Results = new ObservableCollection<Models.Result>();
+
+            SearchMoviesCommand = new Xamarin.Forms.Command(async () =>
```
Comment (arctouch-victorbotamedi) [inline MoviesTips/ViewModels/MoviesSearchViewModel.cs]: This command could be a method, it would be cleaner

**Pull request closed without merging**